hiero-enterprise-base/src/main/java/com/openelements/hiero/base/data/RoyaltyFee.java

```diff
@@ -0,0 +1,14 @@
+package com.openelements.hiero.base.data;
+
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.hashgraph.sdk.TokenId;
+import org.jspecify.annotations.Nullable;
+
+public record RoyaltyFee(
```

Comment:
@Ndacyayisenga-droid we should check if Hedera provides a documentation for those objects and add a javadoc here (we can create an issue for that)

Reply:
> @Ndacyayisenga-droid we should check if Hedera provides a documentation for those objects and add a javadoc here (we can create an issue for that)

There's hedera documentation for all that actually. I will get an issue once this is merged.